index.css

```diff
@@ -1,3 +1,18 @@
-html {
-  
+html {}
+.board { margin:1em auto; border-collapse:collapse }
+.board td {
+    cursor:pointer;
+    width:100px; height:100px;
+    border:5px solid #ccc;
+    text-align:center;
+    font-family:sans-serif; font-size:16px
+}
+
+.board td.x-clicked {
```

Comment:
I think the `td` in here is unnecessary, so I'd leave it out. I prefer not to mix element types and classes together.